pkg/api/frontend_logging.go

```diff
@@ -47,6 +47,18 @@ func (exception *frontendSentryException) FmtStacktraces() string {
 	return strings.Join(stacktraces, "\n\n")
 }
 
+func addEventContextToLogContext(rootPrefix string, logCtx *log15.Ctx, eventCtx *map[string]interface{}) {
+	for key, element := range *eventCtx {
+		prefix := fmt.Sprintf("%s_%s", rootPrefix, key)
+		switch v := element.(type) {
+		case map[string]interface{}:
+			addEventContextToLogContext(prefix, logCtx, &v)
+		default:
+			(*logCtx)[prefix] = fmt.Sprintf("%v", v)
+		}
+	}
+}
```

Comment:
Why should `eventCtx` and `logCtx` be pointers? I don't see why there would be any need, maps are reference types:
```suggestion
func addEventContextToLogContext(rootPrefix string, logCtx log15.Ctx, eventCtx map[string]interface{}) {
	for key, element := range eventCtx {
		prefix := fmt.Sprintf("%s_%s", rootPrefix, key)
		switch v := element.(type) {
		case map[string]interface{}:
			addEventContextToLogContext(prefix, logCtx, &v)
		default:
			logCtx[prefix] = fmt.Sprintf("%v", v)
		}
	}
}
```

Reply:
Fixed! Didn't realize that maps are ref types and are not copied when passed as params. I started reading a book on golang, will get myself up to speed